bitwarden_license/src/Scim/Controllers/v2/UsersController.cs

```diff
@@ -224,23 +224,29 @@ public async Task<IActionResult> Patch(Guid organizationId, Guid id, [FromBody]
             }
 
             var operationHandled = false;
-
-            var replaceOp = model.Operations?.FirstOrDefault(o =>
-                o.Op?.ToLowerInvariant() == "replace");
-            if (replaceOp != null)
+            foreach (var operation in model.Operations)
             {
-                if (replaceOp.Value.TryGetProperty("active", out var activeProperty))
+                // Replace operations
+                if (operation.Op?.ToLowerInvariant() == "replace")
                 {
-                    var active = activeProperty.GetBoolean();
-                    if (active && orgUser.Status == OrganizationUserStatusType.Revoked)
+                    // Active from path
+                    if (operation.Path?.ToLowerInvariant() == "active")
                     {
-                        await _organizationService.RestoreUserAsync(orgUser, null, _userService);
-                        operationHandled = true;
+                        var handled = await HandleActiveOperationAsync(orgUser, operation.Value.GetBoolean());
```

Comment:
If any one of the operation succeeds won't `operationHandled` always end up as true?

Reply:
Yes, that was the goal with this change, which is how the previous behavior worked. We basically want to log the fact that we received a request where no operation was handled.